homeworks/homework_1/task_2/wc.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import os
+from typing import List
+
+
+def print_wordstats(file_names: List[str]):
+    total_lines = 0
+    total_words = 0
+    total_bytes = 0
+
+    for file_name in file_names:
+        try:
+            with open(file_name) as f:
+                file_text = f.readlines()
+                lines = len(file_text)
+                words = sum(len(line.split(" ")) for line in file_text)
+                bytes = os.path.getsize(file_name)
+
+                total_lines += lines
+                total_words += words
+                total_bytes += bytes
+                print(f"{lines} {words} {bytes} {file_name}")
+        except FileNotFoundError:
+            print(f"No such file: {file_name}")
+
+    print(f"{total_lines} {total_words} {total_bytes} итого")
```

Comment:
`wc` actually does not print `total` if only 1 file was input. Can you account for this?